crates/starknet-server/tests/common/utils.rs

```diff
@@ -44,6 +44,20 @@ pub fn load_json<T: serde::de::DeserializeOwned>(path: &str) -> T {
     loaded
 }
 
+pub fn get_flattened_sierra_contract_and_casm_hash(
```

Comment:
function `get_flattened_sierra_contract_and_casm_hash` is identical to `get_events_contract_in_sierra_and_compiled_class_hash` why not leaving only one of them, preferably `get_flattened_sierra_contract_and_casm_hash`

Reply:
Reused the function in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/247/commits/cd8269cce80110f608f0bc5ad7973ca562d7bcd5